### DISCUSSION TITLE
Translation of Open VSX Registry

### DISCUSSION THREAD

**Discussion by yiliang114** [eclipse-theia/theia#11495] [General] [answered]
[https://vscode.dev/github/eclipse-theia/theia/blob/master/packages/core/i18n/nls.zh-cn.json#L384](https://vscode.dev/github/eclipse-theia/theia/blob/master/packages/core/i18n/nls.zh-cn.json#L384)

I think `Open VSX Registry` is [its](https://open-vsx.org/) name and cannot be directly translated `打开VSX注册表` in Chinese. Could it be directly translated into  `应用商店` or `商店` which means `Application Market` in English?  

![image](https://user-images.githubusercontent.com/11473889/182431576-7ea52e08-95ff-4487-9f31-ce18578623b8.png)

![image](https://user-images.githubusercontent.com/11473889/182431333-d6305dd6-7e29-444e-bc41-20694d40333f.png)

**Comment by vince-fugnitto** [comment]
@yiliang114 thank you for the discussion, I'm a little confused by the question, can you clarify?

> I think Open VSX Registry is [its](https://open-vsx.org/) name

That's correct, **Open VSX Registry** is it's name and should not be localized from what I understand.

> Could it be directly translated into 应用商店 or 商店 which means Application Market in English?

Do you mean you'd like the name to be changed to **Application Market** in English? I don't believe this is desirable and translates well.

**Reply by yiliang114** [reply]
I mean there will be no change with the English name `Open VSX Registry`, but to simply modify its translation in Chinese.

Here I have two immature ideas:

1. `打开VSX注册表` amend to `商店` in [core/i18n/nls.zh-cn.json](https://vscode.dev/github/eclipse-theia/theia/blob/master/packages/core/i18n/nls.zh-cn.json#L384). I think The `Open` of  `Open VSX Registry` is more like the meaning of [open-source](https://open-vsx.org/about), but if it is translated into `打开`, it is more like a verb.
2. Or keep `Open VSX Registry` as same as it in English until `Open VSX Registry` has its own official Chinese name.

**Reply by yiliang114** [reply]
This is what I happened to give a little advice, if it doesn't feel good, please ignore it.

**Reply by vince-fugnitto** [reply]
I understand now what you meant, the **Open VSX Registry** is being translated when it should in fact not be.
We do it correctly here:

https://github.com/eclipse-theia/theia/blob/5aeef6c0c683b4e91713ab736957e6655b486adc/packages/vsx-registry/src/browser/vsx-extensions-search-bar.tsx#L44

**Reply by yiliang114** [reply]
> I understand now what you meant, the **Open VSX Registry** is being translated when it should in fact not be. We do it correctly here:
> 
> https://github.com/eclipse-theia/theia/blob/5aeef6c0c683b4e91713ab736957e6655b486adc/packages/vsx-registry/src/browser/vsx-extensions-search-bar.tsx#L44

Yes, this is what I want to express.

**Comment by vince-fugnitto** [comment, marked as answer]
Fixed by https://github.com/eclipse-theia/theia/pull/11523.
